entrypoint.sh

```diff
@@ -2,11 +2,12 @@
 
 # Make sure the folder exists
 mkdir -p /run/weldr
-# Set permissions
-chmod u=rwX,g=rwX,o=--- /run/weldr
+# Set permissions and owership
+chmod u=rwX,g=rX,o=--- /run/weldr
+chown root:weldr /run/weldr
```

Comment:
As discussed on IRC, I'd like to know why we need this in case we need to file a bug against lorax-composer.

In my local testing, lorax-composer always creates this directory with the correct permissions.